config/pitest-suppressions/pitest-xpath-suppressions.xml

```diff
@@ -27,7 +18,7 @@
     <lineContent>switch (axisNumber) {</lineContent>
   </mutation>
 
-  <mutation unstable="true">
```

Comment:
After observing I think the only update i think thats need to be done is https://github.com/checkstyle/checkstyle/pull/13404 https://github.com/checkstyle/checkstyle/actions/runs/5572146155/jobs/10177862384?pr=13404

is i need to make this true again

Reply:
I won't recommend changing any until you have some provenance. This may also be unstable like others. With a few more runs, this might also fail. Please share why you want to do this update.

Reply:
Ok I got it. lets not change the stable and unstable one.